src/List/List.component.js

```diff
@@ -458,10 +457,14 @@ const List = React.createClass({
             hasPreviousPage: () =>
                 Boolean(this.state.pager.hasPreviousPage) && this.state.pager.hasPreviousPage(),
             onNextPageClick: () => {
-                this.setState({ isLoading: true }, () => this.filterList({ page: pager.page + 1 }));
+                this.setState({ isLoading: true }, () =>
+                    this.filterList({ page: this.state.pager.page + 1 })
+                );
             },
             onPreviousPageClick: () => {
-                this.setState({ isLoading: true }, () => this.filterList({ page: pager.page - 1 }));
+                this.setState({ isLoading: true }, () =>
+                    this.filterList({ page: this.state.pager.page - 1 })
+                );
```

Comment:
I guess this change was no really needed for the feature to work, but you wanted to keep a homogeneous usage of `this.state.pager` in all functions, right?